spec/integration/initializer_spec.rb

```diff
@@ -85,6 +99,20 @@
       end.to output("it failed...\n").to_stderr
     end
 
+    it 'returns exit-code 1 and expected appended error-output' do
+      bin_check.check do |config|
+        config.command './spec/spec-files/test_script_exit1'
+        config.error_output '+it failed...'
```

Comment:
Wouldn't the API be nicer to use if the options were called

    config.error_message '.....'

and

    config.success_message '.....'

Also the `+` irritated me a bit. I need to specify a `+` if I want to append output to the commands output, right? What happens if the command itself writes lines with a `+` as the first character of the line?